lib/demos/reference/motion_demo_fade_scale_transition.dart

```diff
@@ -0,0 +1,158 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:animations/animations.dart';
+import 'package:gallery/l10n/gallery_localizations.dart';
+
+// BEGIN fadeScaleTransitionDemo
+
+class FadeScaleTransitionDemo extends StatefulWidget {
+  const FadeScaleTransitionDemo();
+
+  @override
+  _FadeScaleTransitionDemoState createState() =>
+      _FadeScaleTransitionDemoState();
+}
+
+class _FadeScaleTransitionDemoState extends State<FadeScaleTransitionDemo>
+    with SingleTickerProviderStateMixin {
+  AnimationController _controller;
+
+  @override
+  void initState() {
+    _controller = AnimationController(
+      value: 0,
+      duration: const Duration(milliseconds: 150),
+      reverseDuration: const Duration(milliseconds: 75),
+      vsync: this,
+    )..addStatusListener((status) {
+        setState(() {
+          // setState needs to be called to trigger a rebuild because
+          // the 'HIDE FAB'/'SHOW FAB' button needs to be updated based
+          // the latest value of [_controller.status].
+        });
+      });
+    super.initState();
```

Comment:
From the [documentation](https://api.flutter.dev/flutter/widgets/State/initState.html):
> If you override this, make sure your method starts with a call to super.initState().

Reply:
thanks! will change.